### ISSUE TITLE
Karte: Feldkontrollen als WMS-Layer anzeigen

### ISSUE THREAD

**Issue by rebeccakurz** [barbalex/apf2#665]
Kontrollen von vor allem ursprünglichen Teilpopulationen wie die Massnahmen digitalisieren und dann als WMS-Layer in der FloraDB anzeigen. Zuerst digitalisieren (2024?) und dann erst in FloraDB (2025?)

**Comment by barbalex** [comment]
Das gibt bei der FNS mehr Aufwand als hier. Muss vor allem mit ihnen koordiniert werden

**Comment by rebeccakurz** [comment]
Genau, für dieses Jahr haben wir offeriert, dass die Artverantwortlichen uns dazu Shapes abgeben können und wir diese in eine Gesamtdatenbank digitalisieren und dann der FNS abgeben, damit diese einen WMS-Layer daraus machen kann. Gibt erst nächstes Jahr voraussichtlich einen Aufwand für die Datenbank.

**Comment by barbalex** [comment]
Da ich jetzt die Rechnung für das Budget 2025 stellen muss, verfällt dieses. Wir werden diese Massnahme wohl im Budget 2026 wieder aufnehmen müssen. Aber dafür seid ihr zuständig - ich verschanze mir nicht einfach so Aufträge...
